buildkit/pgrouting.yaml

```diff
@@ -7,7 +7,10 @@ source: https://github.com/pgRouting/pgrouting/releases/download/v3.5.1/pgroutin
 description: pgRouting extends the PostGIS/PostgreSQL geospatial database to provide geospatial routing and other network analysis functionality.
 licence: GPL-2.0-or-later
 keywords:
-  - geospatial routing
+  - routing
+  - geospatial
```

Comment:
I would put back `geospatial routing` because each word can't describe the extension but combined can.